### ISSUE TITLE
ARIA dialog and alertdialog nodes should have an accessible name

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8429]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

Ensure every ARIA dialog and alertdialog node has an accessible name
**How to fix:** https://dequeuniversity.com/rules/axe/4.10/aria-dialog-name?application=AxeChrome

![Image](https://github.com/user-attachments/assets/f3139e0f-0874-47a4-af75-8daf6c4548b1)

### Expected behaviour

ARIA dialog and alertdialog nodes should have an accessible name

### How to reproduce

Run axe DevTools
